src/content-helper/dashboard-page/components/posts-table/component.tsx

```diff
@@ -182,6 +192,7 @@ export const PostsTable = ( {
 	className,
 	onPostClick,
 }: PostsTableType ): React.JSX.Element => {
+	// TODO: Add a global state to store the posts for faster loading.
```

Comment:
Should this work be in another PR, or this one?

Reply:
It's optimization work for another PR 🙂